### DISCUSSION TITLE
The 20 proteins for contact prediction training

### DISCUSSION THREAD

**Discussion by Yang-J-LIN** [facebookresearch/esm#105] [Q&A] [answered]
Hi, thanks for the amazing work. I am curious about which 20 proteins are used for the training for unsupervised contact prediction. Could you offer a list of them? And how do you choose these 20 proteins? Select Randomly?

**Comment by tomsercu** [comment, marked as answer]
They were randomly sampled - from the trRosetta train set. https://openreview.net/pdf?id=fylclEqgvgd Section 5.1, title "low-N" describes the sensitivity to the random sample, which is low.
